starfyre/__main__.py

```diff
@@ -9,34 +9,87 @@
 import click
 import importlib.resources as pkg_resources
 
-
 def write_js_file(path):
     dist_path = Path(path) / "dist"
     dist_path.mkdir(exist_ok=True)
     js_store = pkg_resources.path("starfyre.js", "store.js")
     shutil.copy(str(js_store), path + "/dist/store.js")
-    
-def create_main_file(path):
-    """
-    Creates a main file in the build directory.
-    
-    This file will be used to run the project.
-     Every starfyre build will have an `__init__.py` file in the build directory.
-    And the `__init__.py` file will have a component that will render the root component. It will be named `app`.
 
-    You can have a look at the `test-application/build/__init__.py` file to see what it looks like.
+def build_routes_output(generated_routes, path):
+    write_js_file(path)
 
-    The main file is also responsible for adding the `store.js` file
-    """
     output_file_path = path + "/build/__main__.py"
-    write_js_file(path)
+    init_file_path = path + "/build/__init__.py"
+    user_routes = generated_routes[:]
+    user_routes.insert(0, 'app')
+    
+    # create empty __init__.py file
+    with open(init_file_path, "w"):
+        pass
 
     with open(output_file_path, "w") as f:
         f.write(
-"""if __name__ == '__main__':
-    pass
-    """)
+            f'''from starfyre import create_component, render_root
```

Comment:
I wonder if we can have this is a python file somewhere. This string is getting a bit hard to review 😅

Reply:
Okay, I have broken it into another file

Reply:
Amazing, thank you 😄